Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -433,7 +433,20 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
                     // fit into a char?  Why doesn't someone document this?
                     //
                     // --slush
-                    BitStream.Write(static_cast<unsigned char>(pVehicle->GetModel() - 400));
+
+                    unsigned short usModel = pVehicle->GetModel();
+                    BitStream.Write(usModel);
+
```

Comment:
Again, this breaks network compatibility.

Reply:
Use `BitStream.Can` here

Reply:
The problem is that, this way only the clients having this feature will see the actual models, older clients wont. We must break backwards compatibility, but that can't be done now. We should wait for 1.6, where network compatibility will be broken anyways.